lib/matplotlib/colors.py

```diff
@@ -1410,7 +1444,7 @@ def autoscale_None(self, A):
             self.vmax = A.max()
 
     def scaled(self):
-        """Return whether vmin and vmax are set."""
+        """Return whether vim and max are both set"""
```

Comment:
```suggestion
        """Return whether vim and max are both set."""
```

Reply:
Thanks for the very helpful suggestions!